src/Hocon.Configuration/Config.cs

```diff
@@ -18,82 +20,63 @@ namespace Hocon
     ///     configuration string.
     /// </summary>
     [Serializable]
-    public class Config : HoconRoot, ISerializable, IEquatable<Config>
+    public class Config : HoconObject, ISerializable, IEquatable<Config>
```

Comment:
What's the significance of the `HoconRoot` --> `HoconObject` change?

Reply:
Simplified structure, `HoconRoot` does not have any significance, except for passing around substitution list, which is not supposed to be exposed anyway.